### ISSUE TITLE
Will it support network features?

### ISSUE THREAD

**Issue by sinol** [andychase/gbajs2#30]
it seems that this project is so perfect! But is it possible to add some network api that let people link together through websocket? If so, that will be so great to play games like Shinning Souls 2.

**Comment by andychase** [comment]
Yes, I think so! I don't have time to do it but I welcome you looking into it
